rss2email/main.py

```diff
@@ -62,6 +62,9 @@ def run(*args, **kwargs):
     parser.add_argument(
         '-V', '--verbose', default=0, action='count',
         help='increment verbosity')
+    parser.add_argument(
+        '-s', '--set', action='append', nargs=2, metavar=('key', 'value'),
+        help='override config key to value')
```

Comment:
I’m not sure this flag is a good global flag.

What happens with `r2e list`? Does it make sense for `r2e pause`? `unpause`? `oplimport`?

Reply:
The only time I have found this useful so far is when using `r2e add`, so maybe we should just make it a flag for that. I can also envision someone using it with `r2e run` maybe to override the `to` address, to change the `email-protocol` temporarily or to change the mailbox the mail gets sent to, something like that.

Reply:
Let’s just add it to `add` for now and maybe think about other subcommands later.